android/app/src/main/java/app/organicmaps/widget/placepage/EditBookmarkFragment.java

```diff
@@ -267,6 +365,22 @@ private void refreshBookmark()
     refreshColorMarker();
   }
 
+  private void refreshTrack()
+  {
+    if (mTrack == null)
+      return;
+
+    if (TextUtils.isEmpty(mEtName.getText()))
+      mEtName.setText(mTrack.getName());
+
+    if (TextUtils.isEmpty(mEtDescription.getText()))
+    {
+      mEtDescription.setText(BookmarkManager.INSTANCE.getTrackDescription(mTrack.getTrackId()));
```

Comment:
Description is not saved in case of bookmarks so i also didn't saved it in variable. I assumed that may be due to bulky string and processing time to fetch all the description and save in variables it has been avoided thats why i didn't stored them too : )